### PR TITLE
rock paper scissors

### PR DESCRIPTION
**Pull request by kzgodka**

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
+var newGameBtn = document.getElementById('js-newGameButton');
+	
+newGameBtn.addEventListener('click', newGame);
+
+
```
Comment (matwjablonski) [inline js/scripts.js]: remove two empty lines

File: js/scripts.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+function setGamePoints() {
+	playerPointsElem.innerHTML = player.score;
+	computerPointsElem.innerHTML = computer.score;
+
+}
+
+function endGame () {
+	if (player.score == 10) {
```
Comment (matwjablonski) [inline js/scripts.js]: `===` not `==`

File: js/scripts.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+		(computerPick == 'scissors' && playerPick == 'paper') || 
+		(computerPick == 'paper' && playerPick == 'rock')) {
+
+		winnerIs = 'computer';
+	}
+
+	if (winnerIs == 'player') {
+		playerResultElem.innerHTML = "Wygrana!";
+		player.score++;
+	} else if (winnerIs == 'computer') {
```
Comment (matwjablonski) [inline js/scripts.js]: `===` not `==`

File: js/scripts.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+		winnerIs = 'noone'; //remis
+	} else if (
+		(computerPick == 'rock' && playerPick == 'scissors') || 
+		(computerPick == 'scissors' && playerPick == 'paper') || 
+		(computerPick == 'paper' && playerPick == 'rock')) {
+
+		winnerIs = 'computer';
+	}
+
+	if (winnerIs == 'player') {
```
Comment (matwjablonski) [inline js/scripts.js]: `===` not `==`

File: js/scripts.js [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+function checkRoundWinner(playerPick, computerPick) {
+	playerResultElem.innerHTML = computerResultElem.innerHTML = '';
+
+	var winnerIs = 'player';
+
+	if (playerPick == computerPick) {
+		winnerIs = 'noone'; //remis
+	} else if (
+		(computerPick == 'rock' && playerPick == 'scissors') ||
```
Comment (matwjablonski) [inline js/scripts.js]: `===` not `==`

File: js/scripts.js [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+function checkRoundWinner(playerPick, computerPick) {
+	playerResultElem.innerHTML = computerResultElem.innerHTML = '';
+
+	var winnerIs = 'player';
+
+	if (playerPick == computerPick) {
+		winnerIs = 'noone'; //remis
+	} else if (
+		(computerPick == 'rock' && playerPick == 'scissors') || 
+		(computerPick == 'scissors' && playerPick == 'paper') ||
```
Comment (matwjablonski) [inline js/scripts.js]: `===` not `==`

File: js/scripts.js [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+	playerResultElem.innerHTML = computerResultElem.innerHTML = '';
+
+	var winnerIs = 'player';
+
+	if (playerPick == computerPick) {
+		winnerIs = 'noone'; //remis
+	} else if (
+		(computerPick == 'rock' && playerPick == 'scissors') || 
+		(computerPick == 'scissors' && playerPick == 'paper') || 
+		(computerPick == 'paper' && playerPick == 'rock')) {
```
Comment (matwjablonski) [inline js/scripts.js]: `===` not `==`

File: js/scripts.js [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+
+}
+
+function endGame () {
+	if (player.score == 10) {
+		alert("Wygrałeś!")
+		gameState = 'ended';
+		setGameElements();
+
+	}	else if (computer.score == 10) {
```
Comment (matwjablonski) [inline js/scripts.js]: `===` not `==`

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Comment by kzgodka** [comment]
changed '==' to '===' also added reset points after click 'jeszcze raz'

**Comment by kzgodka** [comment]
added new function - reset points and choices

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**